lib/src/video.dart

```diff
@@ -62,7 +66,7 @@ class VideoFrame {
 ///
 class Video extends StatefulWidget {
```

Comment:
Is it possible to add like some accent color like optional parameter to `Video` widget's constructor? I feel just having single color, might not be the best thing for everyone. (If it is easy enough for you to do).